examples/cnab-simple/bundle.json

```diff
@@ -34,7 +34,7 @@
 				"env": "DOCKER_KUBERNETES_NAMESPACE"
 			}
 		},
-		"docker.orchestrator": {
+		"com.docker.app.orchestrator": {
```

Comment:
That's weird I forgot this one... 🤔 
I thought I validated all the examples.

Reply:
I think the rename of this one was missed everywhere, so all the examples and tests worked because there were consistent